colorizer_data/tests/test_converter.py

```diff
@@ -16,34 +38,23 @@ def feature_array_to_dict(
 
 def validate_data(path: pathlib.Path, data: Union[List[int], List[float]]):
     loaded_data = []
-    with open(path, "r") as f:
-        if path.suffix == ".json":
+    if path.suffix == ".json":
```

Comment:
Nit: using pathlib across the board (or not) might help things feel more consistent with all the extension checks

Reply:
Changed! There's definitely areas of the APIs that don't use pathlib.Path though, but that's out of scope for now.